src/ListView/ListView.stories.js

```diff
@@ -0,0 +1,193 @@
+import cx from 'classnames'
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { Row, Col } from 'react-bootstrap'
+import { withKnobs, boolean } from '@storybook/addon-knobs'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+
+import {
+  Button,
+  DropdownKebab,
+  ListView,
+  ListViewInfoItem,
+  ListViewItem,
+  ListViewIcon,
+  MenuItem
+} from '../index'
+import { mockListItems } from './__mocks__/mockListItems'
+
+const stories = storiesOf('ListView', module)
+stories.addDecorator(withKnobs)
+stories.addDecorator(
+  defaultTemplate({
+    title: 'ListView',
+    documentationLink:
+      'http://www.patternfly.org/pattern-library/content-views/list-view/'
+  })
+)
+
+const renderActions = () =>
+  <div>
+    <Button>Details</Button>
+  </div>
+
+const renderAdditionalItems = itemProperties => {
+  return (
+    itemProperties &&
+    Object.keys(itemProperties).map(prop => {
+      const classNames = cx('pficon', {
+        'pficon-flavor': prop === 'hosts',
+        'pficon-cluster': prop === 'clusters',
+        'pficon-container-node': prop === 'nodes',
+        'pficon-image': prop === 'images'
+      })
+      return (
+        <ListViewInfoItem key={prop}>
+          <span className={classNames} />
+          <strong>{itemProperties[prop]}</strong> {prop}
+        </ListViewInfoItem>
+      )
+    })
+  )
+}
+
+stories.addWithInfo(
+  'List of expandable items',
+  `ListView usage example.`,
+  () => {
+    return (
+      <ListView>
+        {mockListItems.map((item, index) =>
+          <ListViewItem
+            key={index}
+            actions={renderActions(item.actions)}
+            checkboxInput={<input type="checkbox" />}
+            leftContent={<ListViewIcon icon="fa fa-plane" />}
+            additionalItems={renderAdditionalItems(item.properties)}
+            heading={item.title}
+            description={item.description}
+            stacked={boolean('Stacked', false)}
+          >
+            <Row>
+              <Col sm={11}>
+                {item.expandedContentText}
+              </Col>
+            </Row>
```

Comment:
I like how the `ListViewItem` knows there is an expansion based solely on if there is a child node rather than requiring an explicit property.